application/src/main/java/uk/gov/hmcts/dm/service/BlobStorageMigrationService.java

```diff
@@ -46,16 +54,20 @@ public BlobStorageMigrationService(CloudBlobContainer cloudBlobContainer, AuditE
     }
 
     public void migrateDocumentContentVersion(@NotNull UUID documentId, @NotNull UUID versionId) {
-
         final DocumentContentVersion documentContentVersion = getDocumentContentVersion(documentId, versionId);
+        migrateDocumentContentVersion(documentContentVersion);
```

Comment:
Is extracting a new method really needed?

Reply:
Got this code from the other branch.  Yes, it is a common method for both individual and batch migration call